openshift-hack/images/os/Dockerfile

```diff
@@ -1,11 +1,9 @@
-# fedora:29 is an image built within the release scripts:
-# https://github.com/openshift/release/blob/b45a09d248b8cdb8fe3bf5f3cfa0b4fee57d04c8/ci-operator/config/openshift/kubernetes/openshift-kubernetes-release-4.10.yaml#L63-L65
-FROM fedora:29 AS build
+FROM quay.io/fedora/fedora:36 AS build
 
 # the registry is defined here:
 # https://github.com/openshift/release/blob/b45a09d248b8cdb8fe3bf5f3cfa0b4fee57d04c8/ci-operator/config/openshift/kubernetes/openshift-kubernetes-release-4.10.yaml#L68
-COPY --from=registry.svc.ci.openshift.org/openshift/origin-v4.0:machine-os-content /srv/ /srv/
-COPY --from=registry.svc.ci.openshift.org/openshift/origin-v4.0:machine-os-content /extensions/ /extensions/
+COPY --from=registry.ci.openshift.org/ocp/4.12:machine-os-content /srv/ /srv/
```

Comment:
One thing I'd highlight here is that the giant pile of hacks we have here *entirely go away* after https://github.com/openshift/enhancements/blob/master/enhancements/ocp-coreos-layering/ocp-coreos-layering.md lands - it can literally be:

```
$ cat Dockerfile
FROM registry.ci.openshift.org/coreos/rhel-coreos:4.12
ADD kubelet /usr/bin
```

And...that's it.  (We need to make the offical tagging real, this relates to https://issues.redhat.com/browse/ART-4352 )

Reply:
That would be such a relief!

Otherwise does that line seem correct to you (as correct as a hack in the pile can be?)